heron/cli/src/python/main.py

```diff
@@ -111,30 +113,21 @@ def run(command, parser, command_args, unknown_args):
   :param unknown_args:
   :return:
   '''
-  status = 1
-  if command == 'activate':
-    status = activate.run(command, parser, command_args, unknown_args)
-
-  elif command == 'deactivate':
-    status = deactivate.run(command, parser, command_args, unknown_args)
-
-  elif command == 'kill':
-    status = kill.run(command, parser, command_args, unknown_args)
-
-  elif command == 'restart':
-    status = restart.run(command, parser, command_args, unknown_args)
-
-  elif command == 'submit':
-    status = submit.run(command, parser, command_args, unknown_args)
-
-  elif command == 'help':
-    status = cli_help.run(command, parser, command_args, unknown_args)
-
-  elif command == 'version':
-    status = version.run(command, parser, command_args, unknown_args)
-
-  return status
-
+  runners = {
+      'activate':activate,
+      'deactivate':deactivate,
+      'kill':kill,
+      'restart':restart,
+      'submit':submit,
+      'update':update,
+      'help':cli_help,
+      'version':version,
+  }
+
+  if command in runners.keys():
+    return runners[command].run(command, parser, command_args, unknown_args)
+  else:
```

Comment:
Good call, will update. This code path will only happen if someone adds a new command to the parser, but doesn't register a runner, so it should get caught in testing, but with the logging it will help pinpoint where the change was missed.

An even better approach would be to somehow have these runners registered as actions with the parser so we don't need this map.